app/services/v2/xccdf_tailoring_file.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module V2
+  # A class representing an XCCDF Tailoring File
+  class XccdfTailoringFile
```

Comment:
As most of this is copy-paste, I'd recommend to revisit this for refactoring after the tests are ready.